src/NuGet.Services.EndToEnd/Support/Clients/V2V3SearchClient.cs

```diff
@@ -172,8 +172,7 @@ public async Task<IReadOnlyList<SearchServiceProperties>> GetSearchServicesAsync
                         searchServices.Add(await GetSearchServiceFromAzureAsync(mappedService, logger));
                     }
                 }
-
-                if (_testSettings.SearchServiceConfiguration.SingleSearchService != null)
+                else
```

Comment:
Does `_testSettings.SearchServiceConfiguration.IndexJsonMappedSearchServices == null` indicate single search? If so, is the `SingleSearchService` config unnecessary?

Reply:
This is actually the meat of the change. I intentionally changed the configuration so that you either test the production slots or the staging slot. Never both.